CHANGE_LOG

```diff
@@ -1,3 +1,113 @@
+Version 0.41.0
+--------------
+
+This release adds the following major features:
+
+* Official ARMv7 (Raspberry Pi 2/3/3+) support - Now building conda packages
```

Comment:
think one of the `np.trace()` tests is failing, preventing this from shipping at present.

Reply:
Good point.  Removed from summary list.